### DISCUSSION TITLE
Magick Identify Compression for AVIF and WEBP

### DISCUSSION THREAD

**Discussion by manuelbabo** [ImageMagick/ImageMagick#7718] [General]
Hi, is there any future plan to expand compression value support for webp and avif image formats for image identification?

**Comment by dlemstra** [comment]
I have no idea what you are trying to ask us?

**Reply by manuelbabo** [reply]
Via the command ‘magick identify’ there is the possibility to see the ‘quality’ value of the image compression. For me this data is important. However, I noticed that it is only returned if the image is in JPEG format and not for AVIF or WEBP.
